src/app/create-resource/create-resource.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import {ProjectService} from '../../services/project-service/project.service';
+import {Resource} from '../interfaces/resource';
+import {ResourceService} from '../../services/resource-service/resource.service';
+import {ActivatedRoute, Params} from '@angular/router';
+import {ToastrService} from 'ngx-toastr';
+
+@Component({
+  selector: 'app-resource',
+  templateUrl: './create-resource.component.html',
+  styleUrls: ['./create-resource.component.css']
+})
+export class CreateResourceComponent implements OnInit {
+  public projectIndex: number;
+  public projectName: string;
+  constructor(private activatedRoute: ActivatedRoute, private projectService: ProjectService,
+              private resourceService: ResourceService, private toastr: ToastrService) { }
+
+  ngOnInit(): void {
+    this.activatedRoute.queryParams.subscribe((params: Params) => {
+      this.projectIndex = params.projectID;
+      this.projectService.getProjects().subscribe(projects => {
+        this.projectName = projects[this.projectIndex].name;
```

Comment:
You shouldn't rely on the index of this array. Also, We tend not to rely on DB ids, but on uuid or some other unique form of identifier, such as names.